### PR TITLE
List SixEleven (611)

### PR DESCRIPTION
**Pull request by fflo**
- Official project URL: https://611project.org && https://github.com/611project/sixeleven
- Official block explorer: https://be.611.to

Link to random transaction (1):
https://be.611.to/block/6d5793cc5f06f4a6748f970cc3871078dce185a75ec2f408f885c23c7a2a76f8
Link to random transaction (2):
https://be.611.to/block/ae9a344381e7f735939df2f0834e00d09c043fd67aa74042b00ced76abe12a38

### CODE REVIEW

**Comment by fflo** [comment]
update Namecoin (NMC) AddressValidator with new Base58BitcoinAddressValidator ...

**Comment by blabno** [comment]
Just one asset per PR please.
Squash the commits into a single one with proper message.

**Comment by fflo** [comment]
ok, everything squashed together into one commit.

**Comment by blabno** [comment]
Just one asset per PR please.

**Comment by fflo** [comment]
Thx.

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 1]
```diff
@@ -137,6 +137,7 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         result.add(new CryptoCurrency("XMR", "Monero"));
         result.add(new CryptoCurrency("NMC", "Namecoin"));
         result.add(new CryptoCurrency("SF", "Siafund"));
+        result.add(new CryptoCurrency("SIL", "SixEleven"));
```
Comment (blabno) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java]: Don't touch this file.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Comment by fflo** [comment]
Thx.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**